src/page-modules/contact/ticket-control/complaint/formMachine.ts

```diff
@@ -0,0 +1,76 @@
+import { assign, setup } from 'xstate';
+
+export const formMachine = setup({
+  types: {
+    context: {} as { title: string },
+  },
+  actions: {
+    setTitle: assign({
+      title: ({ event }) => event.title,
+    }),
+  },
+}).createMachine({
```

Comment:
I think the latest changes, i.e. dividing it into multiple state machines, made this a lot easier to follow along 👍

Reply:
Good to hear!